src/components/Navbar/Navbar.styled.jsx

```diff
@@ -40,35 +41,44 @@ export const NavRight = styled.div`
   justify-content: flex-end;
   align-items: center;
   margin-right: 25px;
-  @media screen and (max-width: 768px) {
+  /* background-color: rgba(191, 255, 0, 0.35); */
+  @media screen and (max-width: 600px) {
+    flex: 5%;
+    margin-right: 0;
+  }
+`;
+
+export const NavSwitch = styled.span`
+  @media screen and (max-width: 600px) {
     display: none;
   }
 `;
 
-export const Input = styled.input`
-  margin-left: 40px;
-  height: 60%;
-  font-size: 12pt;
-  width: 200px;
-  border: none;
-  outline: none;
-  border-radius: 4px;
-  padding: 0 15px;
-  color: #fff;
-  background-color: rgba(255, 255, 255, 0.2);
+// export const Input = styled.input`
```

Comment:
try to avoid commented code, since we're versioning, there is no need of storing old unused code

Reply:
got it